Topological_Sort/Topological_Sort.js

```diff
@@ -0,0 +1,92 @@
+// Program to implement Topological Sort
+class Graph {
+    constructor(nodes) {
+        //2 class variables
+        this.nodes = nodes; // no of vertices
+        this.adj = new Array(nodes); // adjacency list
+        for (let i = 0; i < nodes; i++) {
+            this.adj[i] = new Array();
+        }
+        this.stack = [] // a stack for storing the ending times of dfs
+    }
+
+    showGraph = () => {
+        const {
+            nodes,
+            adj
+        } = this; // destructuring for clarity
+
+        for (let i = 0; i < nodes; i++) {
+            // show graph in readable format
+            console.log(`Vertex ${i} is connected to verices `, adj[i]);
+        }
+    };
+
+    addEdge = (x, y) => {
+        const {
+            nodes,
+            adj
+        } = this;
```

Comment:
Just a suggestion. These lines could have been in one line.

Reply:
@SakshiShreya this format helps in better visualization of the JS array destructuring.